src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Lotto {
+
+    private static final int DRAW_NUMBER = 6;
+
+    private final List<Integer> numbers;
```

Comment:
하나의 로또 번호에 해당하는 `Integer`를 클래스로 포장해보면 어떨까요?
```suggestion
    private final List<LottoNumber> numbers;
```